manifest.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "Blossom",
   "description": "Blossom Fairdata Protocol Browser Extension",
+  "icons": {
+    "48": "icons/blossom48.png",
+    "128": "icons/blossom128.png"
+  },
```

Comment:
I think we need to add `default_icon` to the `action` section as well.

Reply:
Thanks, added this